portafly/portafly_api.yml

```diff
@@ -7,35 +7,55 @@ info:
     A tuned API for PortaFly, a Patternfly SPA consuming 3scale Porta API MGMT solution.
   license:
     name: Apache 2.0
-    url: https://www.apache.org/licenses/LICENSE-2.0
+    url: "https://www.apache.org/licenses/LICENSE-2.0"
 paths:
   /developer/accounts:
     summary: A list of developer accounts
     description: A list of developer accounts.
     get:
       responses:
         "200":
-          $ref: '#/components/responses/developerAccountList'
+          $ref: "#/components/responses/developerAccountList"
       operationId: developerAccountList
       summary: Get developer account list
     parameters:
-    - name: page
-      description: Page in the paginated list. Defaults to 1.
-      schema:
-        type: integer
-      in: query
-      required: false
-    - name: per_page
-      description: Number of results per page. Default and max is 500.
-      schema:
-        type: integer
-      in: query
-      required: false
-    - name: state
-      description: Account state.
-      schema:
-        type: string
-      in: query
+      - name: page
+        description: Page in the paginated list. Defaults to 1.
+        schema:
+          type: integer
+        in: query
+        required: false
+      - name: per_page
+        description: Number of results per page. Default and max is 500.
+        schema:
+          type: integer
+        in: query
+        required: false
+      - name: state
+        description: Account state.
+        schema:
+          type: string
+        in: query
+  /admin/account/users:
```

Comment:
Are we sure the Users depends on Accounts ? even the API endpoint in Porta is not related with Accounts.

Reply:
Also, not sure about the `admin` scope, we didn't scoped it in the first example... is it a reason why this is needed?

Reply:
I used `admin/account` cause I was replicating the url, but you are right is not needed, we can use simply `/users`